src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,11 @@
+.catalog {
+  display: grid;
+  grid-template-columns: repeat(1, 200px);
```

Comment:
The grid-template-columns is set to repeat(1, 200px), which means only one column of 200px will be displayed initially. This might not be optimal for larger screens. Consider using a more flexible unit like 'fr' or 'auto' to allow the grid to adapt better to different screen sizes.